app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -23,3 +23,7 @@ $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 
 // Override other variables below!
+.container {
+  padding-right: 0px!important;
+  padding-left: 0px!important;
+}
```

Comment:
what is this for?

Reply:
to overwrite some padding that bootstrap adds to the container